assets/src/styles/blocks/core-overrides/Image.scss

```diff
@@ -63,14 +63,21 @@
     position: relative;
   }
 
-  .alignleft {
-    margin: 0 $sp-1 $sp-1 0;
-    padding-top: 6px;
+  @include small-and-up {
+    &.alignleft {
+      float: left;
+      margin-inline-end: $sp-1;
+    }
+
+    &.alignright {
```

Comment:
do you know why we do this only for small-and-up ? the alignright ends up on the left on smaller viewport
![Screenshot from 2024-03-11 16-25-12](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/assets/617346/1d6a2b1b-945c-4ca1-bc44-417158ef0bc9)

Reply:
I'm not sure actually, I guessed it was because on mobile it didn't matter since the viewport is smaller... But now that you ask I realise that on posts the viewport is smaller than for other post types, so maybe it makes sense to add it to all screen sizes?

Reply:
According to blame, there was an issue with image captions and float on mobile maybe https://github.com/greenpeace/planet4-master-theme/pull/537 :thinking:

Reply:
I've updated the code so that it gets applied to all screen sizes, seems to work fine to me 🤔